### DISCUSSION TITLE
DFmini busy pin status while playing advertisement

### DISCUSSION THREAD

**Discussion by Schallbert** [Makuna/DFMiniMp3#66] [Q&A]
Hi, just to make sure - could it be that the "busy pin" actually only reports busy when playing a folder track (Mp3FolderTrack) or one of the numbered folders; not when it's currently playing an advertisement?

Thanks.

**Comment by Makuna** [comment]
Note sure myself, asking on their forum would be a good way to find out (https://www.dfrobot.com/forum/viewforum.php?f=8)

**Reply by Schallbert** [reply]
I checked, it doesn't. So:
While it will show the status correctly for "PlayFolderTrack" and "PlayMp3FolderTrack", at least my device does not do so for "PlayAdvertisement".

**Reply by Schallbert** [reply]
Note: What does work is to listen for the player's response via Mp3Notify's OnTrackFinished method, also for Advertisements.

**Reply by seisfeld** [reply]
That heavily depends on the Chip your DFPlayer has and sometimes even on the software version in the Chip.
